docs/CONTRIBUTING.md

```diff
@@ -73,16 +84,17 @@ go: downloading github.com/giantswarm/crd-docs-generator v0.7.1
 Wrote /work/pkg/gadgettracermanager/containers-map/containersmap_bpfel.go
 ```
 
-### Building notes
+### Building Local Gadget
 
-- The compilation uses `tools/image-tag` to choose the tag of the container
-image to use according to the branch that you are compiling.
-- If you wish to make changes to traceloop program, update
-`gadget-default.Dockerfile` to pick your own image of traceloop.
-- As for traceloop, it is also possible to change the BCC to be used as
-described in [BCC](#Updating-BCC-from-upstream) section.
-- You can generate the required BTF information for some well known
-  kernel versions by setting `ENABLE_BTFGEN=true`
+Inspektor Gadget also provides the [`local-gadget`](local-gadget.md) tool as a
+fallback to trace containers when the Kubernetes API server is down. It can be
```

Comment:
I don't think local-gadget is a fallback. It's a different tool to trace containers without Kubernetes.